app/controllers/api/services_controller.rb

```diff
@@ -16,6 +17,15 @@ class Api::ServicesController < Api::BaseController
     actions.sublayout 'api/service'
   end
 
+  def index
+    activate_menu :dashboard
+    @services = current_user.accessible_services
+                            .order(updated_at: :desc)
+                            .paginate(pagination_params)
+                            .decorate
+                            .to_json(only: %i[name updated_at id], methods: %i[links apps_count backends_count unread_alerts_count])
```

Comment:
we should definitely DRY this 😄

Reply:
> TLDR: similar code but no repetition.

Well, I think DRY is not always the best solution: 2 equal lines of code don't necessarily have to be repeated.
In this case and IMO the scopes are different enough (different models, different controllers) so that DRY is not applicable.
Also, today we order by the same column (updated_at), tomorrow we don't know, maybe hits.
As for `paginate`, I wouldn't obscure it behind some decorator but rather leave it explicit for a quicker reading (plus, we'd need to pass the params).
Finally, keep in mind that each to_json passes its own fields and methods.